server/routerlicious/packages/lambdas/src/scriptorium/lambda.ts

```diff
@@ -106,14 +107,24 @@ export class ScriptoriumLambda implements IPartitionLambda {
             ...message,
             mongoTimestamp: new Date(message.operation.timestamp),
         }));
+
+        const documentId = messages[0]?.documentId ?? "";
+        const tenantId = messages[0]?.tenantId ?? "";
+
+        const sequenceNumbers = messages.map((message) => message.operation.sequenceNumber);
+        const sequenceNumberRanges = convertNumberArrayToRanges(sequenceNumbers);
+
         return runWithRetry(
             async () => this.opCollection.insertMany(dbOps, false),
             "insertOpScriptorium",
             3 /* maxRetries */,
             1000 /* retryAfterMs */,
-            getLumberBaseProperties(this.documentId, this.tenantId),
+            { ...getLumberBaseProperties(documentId, tenantId), ...{ sequenceNumberRanges }},
```

Comment:
Simply Nit, this is just unnecessary complex, because it will create two objects, then flat them, then recreate the object again.
If you check `getLumberBaseProperties` definition, it will do:
```

// Helper method to add commonly used Lumber properties
export const getLumberBaseProperties = (documentId: string, tenantId: string) => ({
    [BaseTelemetryProperties.tenantId]: tenantId,
    [BaseTelemetryProperties.documentId]: documentId,
});
```

So you can simply to use an object directly like:
```
{
    [BaseTelemetryProperties.tenantId]: tenantId,
    [BaseTelemetryProperties.documentId]: documentId,
    sequenceNumberRanges
}
```

Reply:
That makes sense @zhangxin511 , however I think there is value in re-using `getLumberBaseProperties` to get the common telemetry properties

Reply:
@shubhi1092 , how about this? Seems cleaner and not involve object deconstruct and reconstruct
```
                    const baseProperties: Record<string, any> = getLumberBaseProperties(documentId, tenantId);
                    baseProperties.sequenceNumberRanges = sequenceNumberRanges;
```

Reply:
Yeah, that could work too. However I feel that the current logic shouldnt be expensive as the objects are really small. Plus its a single line of code vs multiple lines.

Reply:
Hey @zhangxin511, keeping this as is.. but please feel free to unresolve if you feel strongly about changing this. I feel that either approach would work fine here. :)